src/kubelet/metric/cadvisor_test.go

```diff
@@ -49,7 +49,7 @@ func runCAdvisorFetchFunc(t *testing.T, file string) {
 		handler: readerToHandler(f),
 	}
 
-	kubeletClient, err := client.New(client.StaticConnector(c, url.URL{}))
+	kubeletClient, err := client.New(client.StaticConnector(c, url.URL{}), client.WithMaxRetries(3))
```

Comment:
Are the client.WithMaxRetries(3) relevant for the test?

Reply:
can we avoid them? (not sure if it is helpful to test also the client without retries)

Reply:
I removed it